src/app/services/crud.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { CrudService } from './crud.service';
+
+describe('CrudService', () => {
+  let service: CrudService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(CrudService);
+  });
+
+  it('should be created', () => {
```

Comment:
Os nomes das suites de teste devem ser descritivos, devem informar o que estão testando e p que ele espera que aconteça. `should create a service`.
Além disso não sei se há alguma vantagem em testar se a linguagem está funcionando, pois aqui vc só espera que a biblioteca de teste crie a variável. As bibliotecas já tem seus próprios testes, testá-las seria apenas esforço desnecessário. 
Deve-se testar lógica que foi criada, como o método `pegarReceita` do CrudService, por exemplo.

Reply:
Aaa entendi. Mas então, nesse caso foi a própria biblioteca no Angular que gerou esse arquivo de testes. Eu pensei em apagá-lo, visto que eu não usaria agora, mas pensei de deixar para usar futuramente. Acha melhor eu apagar?

Reply:
Entendi. Sim, é melhor apagar, não deixe código inutil na sua base de dados. Mas é bom anotar em algum lugar para ter como referência quando for usar.

Reply:
Beleza, vou fazer isso! Valeu.